airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -548,7 +549,11 @@ def run_query(self,
             expiration as per API specifications. Note that 'field' is not available in
             conjunction with dataset.table$partition.
         :type time_partitioning: dict
-
+        :param cluster_fields: request that the result of the query is stored
+                               sorted by these fields. This can only be used on
+                               a time-partitioned table and must be set when the
+                               table is created. Order is significant.
```

Comment:
Can you fix indent of the above 3 lines? It should be like other multiline comments

Reply:
I think we should add some more details, something along the lines of:

>One or more fields on which data should be clustered in the destination table to save the result of the query. Must be specified with time-based partitioning, data in the table will be first partitioned and subsequently clustered. The order of the specified columns determines the sort order of the data.